stdlib/gc.mli

```diff
@@ -545,3 +545,84 @@ module Memprof :
         Calling [stop] when a callback is running can lead to
         callbacks not being called even though some events happened. *)
 end
+
+(** {1 Debugging memory leaks} *)
+
+external print_reachable : int -> unit = "caml_gc_print_reachable"
+(** [print_reachable min] examines all heap roots and prints
+   the number of words reachable for those that reach at least [min] words.
+   See {!Obj.reachable_words}.
+
+   There are several types of roots.  The most useful ones are
+   global roots, dynamic global roots and stack roots, which can be mapped
+   back to specific variables in the source code.  The filename is
+   easily read from the [print_reachable] output, while {!field_path} provides
+   a way to identify the associated variable.
+
+   The output shows the number of reachable words for each root on the left.
+   To the right is identification information, such as the linker symbol that
+   identifies the source file plus a "field number" that distinguishes
+   global variables defined in the file.
+
+   {b {i Global roots}} correspond to global variables defined in
+   statically-loaded code.  For example, the following line gives the reachable
+   words for a variable defined in the source file [stm.ml]:
+
+   {v    3249706 field  64 camlStm__gc_roots v}
+
+   {b {i Dynamic roots}} correspond to global variables defined in
+   dynamically-loaded code.  For example, the following line gives the
+   reachable words for a variable defined in the source file [ssrvernac.ml]:
+
+   {v     203154 field  19 Ssreflect_plugin__Ssrvernac v}
+
+   {b {i Stack roots}} correspond to local variables in function invocations in
+   the call stack.  For example, the following lines give the reachable words
+   for a particular function invocation.  "[(frame n)]" refers to the {i n}th
+   call stack frame, in which frame 0 is the top of the stack.  Only frames
+   that have roots with the specified minimum number of reachable words are
+   shown.
+
+   {v Called from file "test.ml", line 26, characters 2-22
+      4001 field   1 (frame 0) v}
+
+   The other, less useful, types of roots are: "Global C", "Finalized values",
+   "Memprof" and "Hook".  [print_reachable] isn't able to associate these with
+   specific files.
+
+   [print_reachable] works only when the object code is in ELF format.
+   ELF format is used on Linux systems and Windows/WSL.  Byte-compiled
```

Comment:
```suggestion
   ELF format is used on Linux systems.  Byte-compiled
```
What is there is equivalent to saying "ELF format is used on Linux systems or other systems running Linux in a VM"!"

Reply:
Of course.  The wording is intentional to remind non-WSL Windows users of their ability to use WSL.  WSL gives a far better user experience than non-WSL Windows.  But I can remove if you prefer.

Reply:
The merits or not of WSL are a discussion for another place. The relevant part is that you cannot use WSL to achieve anything useful - you can't build a Windows executable using ELF format, so the mention of Windows is somewhat misleading.